renovate.json5

```
@@ -0,0 +1,105 @@
+// json5: https://json5.org/
+// json5 VS Code extension: https://marketplace.visualstudio.com/items?itemName=mrmlnc.vscode-json5
+// renovate configuration: https://docs.renovatebot.com/configuration-options/
+{
+    $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+
+    extends: [
+        'config:base',
+        ':automergeDisabled',
+        ':combinePatchMinorReleases',
+        ':enableVulnerabilityAlerts',
+        ':enableVulnerabilityAlertsWithLabel(security)',
+        ':label(renovate)',
+        ':rebaseStalePrs',
+        ':semanticCommits',
+        ':semanticPrefixChore',
+        ':timezone(America/Los_Angeles)',
+        ':updateNotScheduled',
+        'workarounds:typesNodeVersioning'
+    ],
+
+    enabledManagers: ['npm'],
+    branchPrefix: 'chore/renovate/',
+    platformCommit: true,
+
+    major: {
+        dependencyDashboardApproval: true
+    },
+    dependencyDashboardAutoclose: true,
+    dependencyDashboardLabels: ['renovate-dashboard'],
+    dependencyDashboardTitle: 'Renovate Dependency Dashboard',
+
+    assigneesFromCodeOwners: true,
+
+    ignorePaths: ['**/templates/**', '**/test-input/**', '**/test-output/**'],
```

```suggestion
    ignorePaths: ['**/templates/**', '**/test-input/**', '**/test-output/**', '**/test/fixtures/**', '**/test/test-data/**'],
```
Looks like this list needs to be extended